auth-web/src/views/auth/staff/ContinuationAuthorizationReview.vue

```diff
@@ -71,31 +71,48 @@
         />
       </v-card>
 
-      <!-- Home Jurisdiction Information -->
+      <!-- Previous Jurisdiction Information -->
       <v-card
-        id="home-jurisdiction-information-vcard"
+        id="previous-jurisdiction-information-vcard"
         flat
         class="mt-8"
       >
         <CardHeader
           icon="mdi-home-city-outline"
-          label="Home Jurisdiction Information"
+          label="Previous Jurisdiction Information"
         />
         <HomeJurisdictionInformation
           :review="review"
           :filing="filing"
         />
       </v-card>
 
+      <!-- Authorization Contact Information -->
+      <v-card
+        id="authorization-contact-information-vcard"
+        flat
+        class="mt-8"
+      >
+        <CardHeader
+          icon="mdi-phone"
+          label="Authorization Contact Information"
+        />
+        <AuthorizationContactInformation
+          :review="review"
+          :filing="filing"
+        />
+      </v-card>
+
       <!-- Continuation Authorization Review Result -->
-      <h2 class="mt-8">
-        Continuation Authorization Review Result
-      </h2>
       <v-card
         id="continuation-authorization-review-result-vcard"
         flat
         class="mt-6"
       >
+        <CardHeader
+          icon="mdi-stamper"
+          label="Authorization Review"
```

Comment:
Per UI Design, we need this icon here:
![image](https://github.com/user-attachments/assets/06bc1dc4-cfc9-4179-8f7f-c69ca1193d2d)

I didn't find this one in MDI, could you please suggest? Thanks a lot

Reply:
This UI uses an older version of MDI fonts. Is `mdi-approval` one of the available icons?

I've also asked Jacqueline for help with this.

Reply:
Thank you! mdi-approval is not available now.

Reply:
Any update on this?

Reply:
It seems this icon is not available now. I didn't find mdi-approval in current or previous version. May I have suggestions?

Reply:
I think what you have now looks OK (even though it looks more like a paint brush than a stamper, haha), but it's up to the UI/UX team to approve this..

Reply:
Haha. Sure, then I will keep this mdi-stamper for now and put a note in the ticket for the UI/UX team to review.

Reply:
Other options might be:
- https://pictogrammers.com/library/mdi/icon/file-check-outline/
- https://pictogrammers.com/library/mdi/icon/text-box-check-outline/
- https://pictogrammers.com/library/mdi/icon/attachment-check/
- https://pictogrammers.com/library/mdi/icon/checkbox-multiple-outline/

cc: @jacqueline-williams-549